src/main/sql/create.sql

```diff
@@ -1,5 +1,5 @@
-create table "COMPUTERS" ("ID" INTEGER NOT NULL,"NAME" VARCHAR NOT NULL,"MANUFACTURER_ID" INTEGER NOT NULL);
-create table "COMPANIES" ("ID" INTEGER NOT NULL,"NAME" VARCHAR NOT NULL);
+create table IF NOT EXISTS "COMPUTERS" ("ID" INTEGER NOT NULL,"NAME" VARCHAR NOT NULL,"MANUFACTURER_ID" INTEGER NOT NULL);
+create table IF NOT EXISTS "COMPANIES" ("ID" INTEGER NOT NULL,"NAME" VARCHAR NOT NULL);
```

Comment:
why is the IF NOT EXISTS needed? this creates an in-memory db every time, right?